app/controllers/calendar_controller.rb

```diff
@@ -13,9 +13,9 @@ class InvalidCalendarScope < StandardError; end
   def show_calendar
     respond_to do |format|
       format.html do
-        @faq_presenter = FaqPresenter.new(scope, calendar, content_item_hash, view_context)
+        @faq_presenter = FaqPresenter.new(Calendar.file_name(params[:scope]), calendar, content_item_hash, view_context)
```

Comment:
Why `FaqPresenter` being instantiated with `Calendar.file_name(params[:scope])` now rather than `calendar`? Are they different?

Reply:
Fixed to return calendar.scope